tests/test_utils_unit.py

```diff
@@ -111,14 +111,14 @@ def singularity_test_data():
     return [SingularityTestCase(**test_json) for test_json in tests_json]
 
 
-def site_blacklist_test_data(good=True):
-    """Pull in site/blacklist test data from data file and return a list of
+def site_blocklist_test_data(good=True):
+    """Pull in site/blocklist test data from data file and return a list of
     test cases"""
     SiteAndBlacklistTestData = namedtuple(
```

Comment:
We should probably change this to `SiteAndBlocklistTestData`, as well as the return value of this function

Reply:
So yes, getting those...
There's also a +Blacklist_Sites classad, if we change that do we have to change monitoring, etc.?

Reply:
Yeah - I think we should keep those as is, because that's the infrastructure side, and may have unforeseen downstream consequences.